api/tests/integration/application/certifications/index_test.js

```diff
@@ -35,7 +41,25 @@ describe('Integration | Application | Route | Certifications', function() {
       return promise.then(response => {
         expect(response.statusCode).to.equal(200);
       });
+    });
+  });
+
+  describe('PATCH /api/certifications/:id', () => {
+
+    it('should exist', () => {
```

Comment:
on a dit qu'on pouvait supprimer les tests squelette de type “it should exists”. Donc tu peux supprimer ça